airflow/www/security_manager.py

```diff
@@ -348,3 +348,9 @@ def _is_authorized_category_menu(self, category: str) -> Callable:
             self._get_auth_manager_is_authorized_method(fab_resource_name=item)(action, resource_pk, user)
             for item in items
         )
+
+    def add_permissions_view(self, base_action_names, resource_name):
```

Comment:
These methods were removed in #37889 but they are actually needed

Reply:
Should these be added back in a separate PR?

Reply:
I can do that if you prefer, I dont have strong opinion on that one. But yeah, that would make sense. I can do that

Reply:
#37997

Reply:
I converted this PR as a draft, waiting for #37997 to be merged first. I'll rebase this PR and put it back in review mode

Reply:
Done